.github/workflows/ci.yml

```diff
@@ -31,8 +31,6 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2.3.3
-        with:
-          submodules: recursive
```

Comment:
Removed because restoring submodules is part of `dotnet build`;

https://github.com/getsentry/sentry-unity/pull/174/files#diff-d77672cdc87b873cf5dd4359e72764340e39d697cb8100c88e1363ae5ff2c67a